examples/autokey/autokey-setup/README.md

```diff
@@ -0,0 +1,25 @@
+# Autokey Example
```

Comment:
`/autokey-setup` and `/keyhandle-setup` examples are not being executed by CI because of the current folder structure (`examples/autokey/autokey_...`).
The [blueprint-test framework](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test) expect the examples to be directly behind `examples/` folder. (At least using default configs. It's possible to override that with some changes in test files. [Here's a reference.](https://github.com/terraform-google-modules/terraform-example-foundation/blob/aca34008b41e4dc16977c22130b785f015f4c9ca/test/integration/envs/envs_test.go#L33))

Reply:
thanks for the info :)

moved the folders inside /examples directly and gave them relevant name.
Please take a look